apps/src/templates/teacherDashboard/SectionRow.jsx

```diff
@@ -94,7 +94,8 @@ ConfirmDelete.propTypes = {
 
 const ProviderManagedSectionCode = ({provider}) => (
   <div data-tip={i18n.providerManagedSection({provider})}>
-    {i18n.none()}
```

Comment:
It looks like the `none` localization is no longer used anywhere.  Let's remove it.